collections/_article/godot-game-awards-stayathomedev.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Meet the Creator Behind the Godot Game Awards"
+excerpt: "StayAtHomeDev has been sharing his passion in YouTube for over two years and now he's running the awards."
+categories: ["showcase"]
+author: Emi
+image: /storage/blog/covers/godot-awards-stayathomedev.webp
+date: 2024-12-09 15:00:00
+---
+
+If you’ve watched Godot-related content on YouTube, [StayAtHomeDev](https://www.youtube.com/@stayathomedev) needs no introduction. He’s been sharing his passion for game development and the Godot Engine through his channel. As the creator of the the popular “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, and the Godot Game Awards, he’s helped showcase the amazing work of the Godot community. We had the chance to talk with him about his journey, what inspired the awards, and his efforts to support the community.
```

Comment:
```suggestion
If you’ve watched Godot-related content on YouTube, [StayAtHomeDev](https://www.youtube.com/@stayathomedev) needs no introduction. He’s been sharing his passion for game development and the Godot Engine through his channel. As the creator of the popular “[This Week in Godot](https://www.youtube.com/playlist?list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, and the Godot Game Awards, he’s helped showcase the amazing work of the Godot community. We had the chance to talk with him about his journey, what inspired the awards, and his efforts to support the community.
```
I think this link is preferable as it links to the playlist itself, rather than to one particular video within it with the playlist shown alongside.